src/main/app/forms/validation/validators/dateFutureConstraint.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+
+import { MomentFactory } from 'common/momentFactory'
+import { LocalDate } from 'forms/models/localDate'
+
+@ValidatorConstraint()
+export class DateFutureConstraint implements ValidatorConstraintInterface {
+  validate (value: any, args?: ValidationArguments) {
+    if (value === undefined) {
```

Comment:
If value can be undefined it means in fact it was not passed to this method. In that case should this not be like:

```
validate (value?: any
```

or it breaks the interface?

Reply:
it does break the interface